src/server_manager/web_app/ui_components/outline-gcp-oauth-step.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright 2021 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import '@polymer/polymer/polymer-legacy.js';
+import '@polymer/iron-pages/iron-pages.js';
+import '../ui_components/outline-step-view.js';
+
+import {css, customElement, html, LitElement, property} from 'lit-element';
+import {COMMON_STYLES} from '../ui_components/cloud-install-styles';
+
+@customElement('outline-gcp-oauth-step')
+export class GcpConnectAccountApp extends LitElement {
+  @property({type: Function}) onCancel: Function;
+  @property({type: Function}) localize: Function;
+
+  static get styles() {
+    return [
+      COMMON_STYLES, css`
+      :host {
+      }
+      .container {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        height: 100%;
+        align-items: center;
+        padding: 132px 0;
+        font-size: 14px;
+      }
+      #connectAccount img {
+        width: 48px;
+        height: 48px;
+        margin-bottom: 12px;
+      }
+      .card {
+        display: flex;
+        flex-direction: column;
+        align-items: stretch;
+        justify-content: space-between;
+        margin: 24px 0;
+        padding: 24px;
+        background: var(--background-contrast-color);
+        box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.14), 0 2px 2px 0 rgba(0, 0, 0, 0.12), 0 1px 3px 0 rgba(0, 0, 0, 0.2);
+        border-radius: 2px;
+      }
+      @media (min-width: 1025px) {
+        paper-card {
+          /* Set min with for the paper-card to grow responsively. */
+          min-width: 600px;
+        }
+      }
+      .card p {
+        color: var(--light-gray);
+        width: 100%;
+        text-align: center;
+      }
+      .card paper-button {
+        color: var(--light-gray);
+        width: 100%;
+        border: 1px solid rgba(255, 255, 255, 0.12);
+        border-radius: 2px;
+      }
+      .card paper-button[disabled] {
+        color: var(--medium-gray);
+        background: transparent;
+      }
+      /* Mirror images */
+      :host(:dir(rtl)) .mirror {
```

Comment:
I don't think this will work. dir() doesn't exist, it's a Polymer and Firefox thing.
I think you'll need to set the dir attribute and do `:host[dir=rtl]`

Reply:
We would need to run the RTL CSS processor on the LitElement components if we wanted to keep writing the styling the same way.

It may be easier to just do what you suggest.